src/views/token-swap/components/amount-box/amount-box.view.jsx

```diff
@@ -95,31 +69,49 @@ function AmountBox ({
             placeholder='0'
             onChange={onInputChange}
           />
-          {renderDropdown()}
         </div>
         <div className={`${classes.row} ${classes.rowMarginTop}`}>
-          {renderBalance()}
+          {account
+            ? (
+              <p className={classes.convertedText}>
+                Balance: {`${balance} ${account.token.symbol} `}
+                {position === AmountBoxPosition.FROM && (
+                  <span className={classes.maxBtn} onClick={setMax}>
+                    Max
+                  </span>
+                )}
+              </p>
+              )
+            : <div />}
```

Comment:
Is this `div` element needed? Maybe we can just do `account && <p className={...`?